src/main.py

```diff
@@ -219,6 +219,7 @@ def _generate_interaction_results(
             request.train_task_idx,
             request.termination_function,
             max_num_steps=CFG.max_num_steps_interaction_request,
+            exceptions_to_break_on={utils.OptionExecutionFailure},
```

Comment:
are there any others using online stuff that this might affect? @tomsilver

Reply:
yeah Ashay actually added this line too in his open PR, for the same reason as we're adding it here